src/workspace.js

```diff
@@ -1065,6 +1065,12 @@ module.exports = class Workspace extends Model {
         if (typeof item.setCursorBufferPosition === 'function') {
           item.setCursorBufferPosition([initialLine, initialColumn])
         }
+        if (typeof item.unfoldBufferRow === 'function') {
+          item.unfoldBufferRow(initialLine)
+        }
+        if (typeof item.scrollToBufferPosition === 'function') {
+          item.scrollToBufferPosition([initialLine, initialColumn], { center: true })
```

Comment:
I've included a test to verify the unfolding behavior, but I have not included a test for scrolling the line to the center of the editor.

After looking into what it would take to add a test for scrolling the line to the center of the editor, it _seems_ like it the effort required to add and maintain that test probably isn't worth the rather minor benefit that we'd get from such a test. I think we'd have to set the window to a certain size and then check the coordinates of the line in the viewport. I think it's do-able, but the benefit doesn't feel like it's worth the effort. If anyone feels differently and has thoughts on how to best test this change, please let me know.